internal/cli/atlas/quickstart/prompt.go

```diff
@@ -89,3 +95,31 @@ func newClusterProviderQuestion() *survey.Question {
 		},
 	}
 }
+
+func newMongoShellQuestion(clusterName string) *survey.Confirm {
+	return &survey.Confirm{
+		Message: fmt.Sprintf("Do you want to access %s with MongoDB Shell?", clusterName),
+		Help:    mongoshHelp,
+	}
+}
+
+func newMongoShellQuestionProvidePath() *survey.Confirm {
+	return &survey.Confirm{
+		Message: "No MongoDB shell version detected. Do you want to provide the path to your MongoDB shell binary?",
+		Help:    mongoshNotFoundHelp,
+	}
+}
+
+func newMongoShellPathInput(defaultValue string) survey.Prompt {
+	return &survey.Input{
+		Message: "Default MongoDB Shell Path:",
+		Help:    mongoshHelp,
+		Default: defaultValue,
+	}
+}
+
+func newMongoShellQuestionOpenBrowser() *survey.Confirm {
+	return &survey.Confirm{
+		Message: "Do you want to download MongoDB Shell?",
```

Comment:
from the demo yesterday it looks like this will launch the browser, can we add a note here?
```suggestion
		Message: "Do you want to download MongoDB Shell (this will open www.mongodb.com on your browser)?",
```
Or something like it